source/vibe/db/redis/redis.d

```diff
@@ -417,6 +417,15 @@ struct RedisDatabase {
 		else return request!(RedisReply!T)("ZRANGE", key, start, end);
 	}
 
+	/// When all the elements in a sorted set are inserted with the same score, in order to force lexicographical ordering, 
+	/// this command returns all the elements in the sorted set at key with a value between min and max.
+	RedisReply!T zrangeByLex(T = string)(string key, string min = "-", string max = "+", long offset = 0, long count = -1)
+		if(isValidRedisValueType!T)
+	{
+		if (offset > 0 || count != -1) return request!(RedisReply!T)("ZRANGEBYLEX", key, min, max, "LIMIT", offset, count);
+		else return request!(RedisReply!T)("ZRANGEBYLEX", key, min, max);
+	}
+
```

Comment:
Change looks unrelated. I know it's related to http://forum.rejectedsoftware.com/groups/rejectedsoftware.vibed/thread/24144/ but it should be in a separate P.R.